cmd/workspace/workspace-bindings/workspace-bindings.go

```diff
@@ -146,7 +146,7 @@ func newGetBindings() *cobra.Command {
   or an owner of the securable.
 
   Arguments:
-    SECURABLE_TYPE: The type of the securable to bind to a workspace.
+    SECURABLE_TYPE: The type of the securable to bind to a workspace. Supported values: [catalog, credential, external_location, storage_credential] 
```

Comment:
Why is it lower cased here?

Reply:
This is lowercased in the OpenAPI spec: https://docs.databricks.com/api/workspace/workspacebindings/updatebindings#securable_type